### PR TITLE
Make UserSettingsPage fit on a phone screen

### PR DESCRIPTION
**Pull request by maltee1**
I decided to only change the bare minimum and keep cosmetic changes for another time.
What I changed:
- remove margins
- put the ScrollView below the back button in collapsed mode
- allow labels to wrap

This does affect the layout in non-collapsed mode, because the right and bottom margins are gone. They didn't look good anyway.

### CODE REVIEW

File: resources/qml/pages/UserSettingsPage.qml [hunk 1]
```diff
@@ -25,7 +25,9 @@ Rectangle {
         palette: Nheko.colors
         ScrollBar.horizontal.visible: false
         anchors.fill: parent
-        anchors.margins: Nheko.paddingLarge
+        //anchors.margins: Nheko.paddingLarge // padding right of the scrollbar is ugly anyway
+        anchors.topMargin: (collapsed? backButton.height : 0) + Nheko.paddingMedium
+        anchors.leftMargin: collapsed? Nheko.paddingMedium : Nheko.paddingLarge
```
Comment (deepbluev7) [inline resources/qml/pages/UserSettingsPage.qml:30]: You are missing the bottom padding. That makes scrolling down weird.

**Comment by deepbluev7** [comment]
Screenshot_20220220_025245

I think those dropdowns need a max width of like 400 or so. At least less than half of the collapse point.

File: resources/qml/pages/UserSettingsPage.qml (hunk 1 above)
Comment (deepbluev7) [reply resources/qml/pages/UserSettingsPage.qml:30]: image

Comment (maltee1) [reply resources/qml/pages/UserSettingsPage.qml]: I didn't want the borders, but some distance of the last item from the bottom makes sense.

**Pull request merged**